draft-ietf-tls-esni.md

```diff
@@ -1061,6 +1061,11 @@ As above, the payload of "encrypted_client_hello" is expected to be a
 `ClientECH` with `ClientECH.type` is `inner`. If this is not the case, the
 backend server MUST abort the handshake with an "illegal_parameter" alert.
 
+The backend server does not confirm ECH acceptance twice. In particular, if it
+confirmed acceptance in HelloRetryRequest as described in this section, it does
+not confirm acceptance in the subsequent ServerHello. Instead, it computes the
+ServerHello in the usual way.
```

Comment:
I suppose either works, but is there a reason we want to do it this way over just having it be confirmed in both? (I figure both reduces the number of variations between HRR and non-HRR SH.)

Reply:
Though I suppose both means the client needs to check it.

Reply:
It seems like confirming in both is easier to implement. The only downside I see is that confirming in both requires a bit more hashing.

Reply:
I'm fine either way, I just think this should be made explicit.

Reply:
It does seems simpler to do both (for client and server), and that has been my assumption here. I'd prefer if we did that.